qcelemental/_version.py

```diff
@@ -202,15 +199,15 @@ def git_versions_from_keywords(keywords, tag_prefix, verbose):
     for ref in sorted(tags):
         # sorting will prefer e.g. "2.0" over "2.0rc1"
         if ref.startswith(tag_prefix):
-            r = ref[len(tag_prefix):]
+            r = ref[len(tag_prefix) :]
             if verbose:
                 print("picking %s" % r)
             return {
                 "version": r,
                 "full-revisionid": keywords["full"].strip(),
                 "dirty": False,
                 "error": None,
-                "date": date
+                "date": date,
```

Comment:
wait, this file is external and should never be touched!

Reply:
We ended up just formatting it first with YAPF and now with black.